recon_surf/README.md

```diff
@@ -20,10 +20,10 @@ List them by running the following command:
 ### Required arguments
 * --sd: Output directory \$SUBJECTS_DIR (equivalent to FreeSurfer setup --> $SUBJECTS_DIR/sid/mri; $SUBJECTS_DIR/sid/surf ... will be created).
 * --sid: Subject ID for directory inside \$SUBJECTS_DIR to be created ($SUBJECTS_DIR/sid/...)
-* --t1: T1 full head input (not bias corrected). 
+* --t1: T1 full head input (not bias corrected). This must be conformed (dimensions: 256x256x256, voxel size: 1x1x1, LIA orientation, and data type UCHAR). Images can be conformed using FastSurferCNN's [conform.py](https://github.com/Deep-MI/FastSurfer/blob/master/FastSurferCNN/data_loader/conform.py) script.
 
 ### Optional arguments
-* --seg: Global path with filename of segmentation (where and under which name to find it, must already exist). Default location: $SUBJECTS_DIR/$sid/mri/aparc.DKTatlas+aseg.deep.mgz
+* --seg: Global path with filename of segmentation (where and under which name to find it, must already exist). This must be conformed (dimensions: 256x256x256, voxel size: 1x1x1, and LIA orientation). FastSurferCNN's segmentations are conformed by default. Please ensure that segmentations produced otherwise are also conformed. Default location: $SUBJECTS_DIR/$sid/mri/aparc.DKTatlas+aseg.deep.mgz
```

Comment:
Also point to conform.py and default command (like it is done in recon-surf.sh argument description)

Reply:
Same as the similar comment: I think conform.py currently doesn't support conforming segmentations.